### PR TITLE
Add script to calculate mean, stdev for continuous 3D field parameter and facies probabilities for 3D facies parameter per zone

### PR DESCRIPTION
**Pull request by oddvarlia**
…s and estimated facies probabilities for discrete 3D parameter

### CODE REVIEW

**Comment by oddvarlia** [comment]
Purpose:
Calculate ensemble mean and standard deviation for 3D continuous parameters and estimates for facies probabilties for 3D facies parameters.
The estimates should be available per zone in the geomodel grid also for ensembles where the 3D geomodel grid vary from realization to realization as long as number of grid cells laterally (nx, ny) is fixed. To handle this, a fixed "gox" grid called ERTBOX grid is used.
The plan is also to be able to calculate this for field parameters used in APS in AHM workflows in ERT using ES-MDA:

**Comment by oddvarlia** [comment]
I will add a specification using yaml as config file format instead of directly specifying a python dict.

File: pyproject.toml
Comment (alifbe) [inline pyproject.toml:1]: I think you also need to register the workflow job in `[project.entry-points.ert]` section.

File: src/subscript/field_statistics/FIELD_STATISTICS
Comment (alifbe) [inline src/subscript/field_statistics/FIELD_STATISTICS:1]: I believe you don't need this file with the way it is set-up

File: src/subscript/field_statistics/field_param_stat.yml
Comment (alifbe) [inline src/subscript/field_statistics/field_param_stat.yml:1]: Is this file still needed?

File: src/subscript/field_statistics/wf_field_statistics
Comment (alifbe) [inline src/subscript/field_statistics/wf_field_statistics:1]: I believe this file is not needed anymore?

File: tests/testdata_field_statistics/ert/model/0readme
Comment (alifbe) [inline tests/testdata_field_statistics/ert/model/0readme:1]: Does the script required this folder structure? Is that the reason for having this 0readme file?

**Comment by alifbe** [comment]
I think this PR looks great. I am still not sure if you need to have `ert/model` in your testdata folder. 

Perhaps @rnyb  or @mferrera  can also have a look at this PR?

File: pyproject.toml
Comment (oddvarlia) [reply pyproject.toml:1]: Done

File: src/subscript/field_statistics/FIELD_STATISTICS
Comment (oddvarlia) [reply src/subscript/field_statistics/FIELD_STATISTICS:1]: Is now removed

File: src/subscript/field_statistics/field_param_stat.yml
Comment (oddvarlia) [reply src/subscript/field_statistics/field_param_stat.yml:1]: Is now removed

File: src/subscript/field_statistics/wf_field_statistics
Comment (oddvarlia) [reply src/subscript/field_statistics/wf_field_statistics:1]: Is now removed

File: tests/testdata_field_statistics/ert/model/0readme
Comment (oddvarlia) [reply tests/testdata_field_statistics/ert/model/0readme:1]: The command line input take the CONFIG_PATH for ERT as one of the input, and the code read the global_variables.yml file from fmuconfig/output to get realization independent info about facies per zone and for that purpose I use the ERT CONFIG PATH and therefore use this directory structure in the testdata

**Review by mferrera — APPROVED** [review]
There are a few improvements I think are possible here (print statements, commented out code, etc) but as a first pass to initialize a script I think we can take this as is for now.
